Vagrantfile

```diff
@@ -219,4 +219,4 @@ Vagrant.configure(2) do |config|
     config.ssh.password = 'vagrant'
     config.ssh.insert_key = 'true'
   end
-end
```

Comment:
Un-needed change

Reply:
Hey, I'm not sure how to remove this change, I don't want to change the vagrant file, and I didn't even realize I was changing it (my bad). Also is it a problem that this PR has all of the previous commits?